queue/queue.py

```diff
@@ -1,17 +1,38 @@
 import sys
 sys.path.append('../linked_list')
-from linked_list import LinkedList
+from linked_list import LinkedList, Node
 
 class Queue:
   def __init__(self):
     self.size = 0
     self.storage = LinkedList()
 
   def enqueue(self, item):
-    pass
+    new_node = self.storage.tail
+    # print(new_node)
+    if self.storage.head == None:
+       self.storage.head = item 
+    else:
+      new_node = item
+    new_node = item
+
```

Comment:
We've worked on this together, but as a reminder, your `LinkedList` object has various methods you defined before, such as `add_to_tail()`. Use these tools to your advantage when you implement functionality in queue.